### PR TITLE
Open CFP and OG pages for public viewing

### PR DESCRIPTION
**Pull request by drewbrew**
## Description

1. Refreshes the OG and speaking pages for 2024
2. Adds a link to both from the home page
3. Refreshes the speaker resources page
4. Removes a couple of links (COVID policy, Sponsor info) until those pages are ready

### CODE REVIEW

**Comment by drewbrew** [comment]
# Screenshots

## home
homepage
## OG
opportunity-grants
## CFP
speaking
## VIsas
visas
## Speaker resources

speaker-resources

**Review by jefftriplett — APPROVED** [review]
LGTM. I double-checked the OG form too.

**Review by emullaney — APPROVED** [review]
Looks good to me, as well!

**Review by PeterGrand — APPROVED** [review]
Nice job!

File: _pages/opportunity-grants.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ We require receipts for the items that the opportunity grant was spent on.
 
 If you are traveling into the United States, we strongly encourage you to wait to purchase airfare or book a hotel room until after your visa has been accepted.
 
-You can request a visa letter at <a href="/invitation">Get Invitation Letter</a>.
+Find out more details at our <a href="/visas/">Visa information page</a>.
```
Comment (tim-schilling) [inline _pages/opportunity-grants.md]:
```suggestion
Find out more details and get your invitation letter at our <a href="/visas/">Visa information page</a>.
```
If someone is looking for the invitation letter, they may be looking for that specific text. Being clear should help folks in that scenario.

**Pull request merged**